test/jdk/java/nio/channels/DatagramChannel/StressNativeSignal.java

```diff
@@ -103,11 +116,21 @@ public void terminate() {
                 // ignore
             }
         }
+
+        public void waitTestThreadStart() {
+            try {
+                threadStarted.await();
+            } catch (InterruptedException intEx) {
+                System.err.println("ServerSocketThread.waitTestThreadStart:" +
+                                   " InterruptException caught ... continue");
```

Comment:
Why catch InterruptedException here?

Reply:
I don't quite get the innuendo  .... while replacing the Thread.sleep with a countdown latch, we're emulating what the existing code is doing ... Thread sleep catches any exception and continues ... same idiom was applied here